src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests.Android/Resources/layout/activity_main.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this? Maybe if it came from Essentials then it was just never removed.